### PR TITLE
hints for revegetate, bronze

### PR DESCRIPTION
**Pull request by ryanchou-dev**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/bronze/usaco-916.mdx [hunk 1]
```diff
@@ -6,7 +6,12 @@ author: Ryan Chou, Brad Ma
 ---
 ## Solution 1
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_revegetate_bronze_feb19.html)
+<Spoiler title="Hint 1">
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-916.mdx:9]: if it's just 1 hint than just "Hint"

Comment (ryanchou-dev) [reply solutions/bronze/usaco-916.mdx:9]: oops, thanks

File: solutions/bronze/usaco-916.mdx [hunk 2]
```diff
@@ -6,7 +6,12 @@ author: Ryan Chou, Brad Ma
 ---
 ## Solution 1
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_revegetate_bronze_feb19.html)
+<Spoiler title="Hint">
 
+What's the best case scenario, and what forces us to change to a different grass type?
+
+</Spoiler>
+<Spoiler title="Solution">
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-916.mdx:14]: break solution 1 & solution 2 into their separate spoilers

File: solutions/bronze/usaco-916.mdx [hunk 3]
```diff
@@ -73,13 +76,15 @@ public class Revegetation {
 </JavaSection>
 	
 </LanguageSection>
+</Spoiler>
+
+<Spoiler title="Solution 2">
 
-## Solution 2
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-916.mdx:72]: replace this w/ ## Explanation

File: solutions/bronze/usaco-916.mdx [hunk 4]
```diff
@@ -4,11 +4,22 @@ source: USACO Bronze 2019 February
 title: The Great Revegetation
 author: Ryan Chou, Brad Ma
 ---
-## Solution 1
+
+<Spoiler title="Hint">
+
+What's the best case scenario, and what forces us to change to a different grass type?
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-916.mdx:10]: wdym "best case scenario"

Comment (ryanchou-dev) [reply solutions/bronze/usaco-916.mdx:10]: a case where everyone can share the same grass, so you only have 1111...

Comment (SansPapyrus683) [reply solutions/bronze/usaco-916.mdx:10]: yeah, imo the first part (before the comma) isn't rlly necessary?

Comment (ryanchou-dev) [reply solutions/bronze/usaco-916.mdx:10]: ok bet

**Pull request merged**